docs/JMAQS_1/appium/AppiumDriverFactory.md

```diff
@@ -1,114 +1,59 @@
-# <img src="resources/maqslogo.ico" height="32" width="32"> AppiumDriverFactory
-The Appium Driver Factory class is a utility class for working with Selenium.
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Appium Driver Factory
 
-## Inheritance Hierarchy
-```java
-com.magenic.jmaqs.appium.AppiumDriverFactory
-```
-Package: com.magenic.jmaqs.appium;
-Assembly: import com.magenic.jmaqs.appium.AppiumDriverFactory
-
-## Syntax
-Java
-```java
-public class AppiumDriverFactory
-```
+## Overview
+Takes care of the base test context data.
```

Comment:
Not an accurate overview

Reply:
@jason-edstrom  review needed for updated overview